kafka-java-console-sample/docs/Cloud_Foundry.md

```diff
@@ -16,7 +16,7 @@ To build and run the sample, you must have the done the following:
 
 By this point, you should have an Event Streams for IBM Cloud instance provisioned. If you haven't done this step yet, please refer to the main [readme](../README.md).
 
-If you are not sure what type of Event Streams for IBM Cloud instance you have then you can find this information out by visiting IBM Cloud's web console [dashboard](https://console.bluemix.net/dashboard).
+If you are not sure what type of Event Streams for IBM Cloud instance you have then you can find this information out by visiting IBM Cloud's web console [dashboard](https://cloud.ibm.com/resources).
```

Comment:
choose between the two plan: https://cloud.ibm.com/docs/services/EventStreams?topic=eventstreams-plan_choose